examples/docker-mixin-test/porter.yaml

```diff
@@ -37,6 +43,14 @@ install:
       command: cowsay
       arguments:
         - "Hello World"
+- docker:
+    description: "Login to docker"
+    login:
+- docker:
```

Comment:
Are your pushes working now that we have login?

Reply:
Yes!